website/content/docs/configuration/controller.mdx

```diff
@@ -35,18 +35,35 @@ description will be read.
     Boundary will not be able to connect by default. To run Boundary without a TLS connection
     to Postgres (not recommended for production usage), add the `sslmode=disable` parameter to
     your connection string, such as `url = "postgresql://postgres:boundary@192.168.1.1:5432/boundary?sslmode=disable"`
+    This value can refer to a file on disk (file://) from which a URL will be read; an env
+    var (env://) from which the URL will be read; or a direct database URL (postgres://).
   - `migration_url` - Can be used to specify a different URL for migrations, as that
     usually requires higher privileges.
+    This value can refer to a file on disk (file://) from which a URL will be read; an env
+    var (env://) from which the URL will be read; or a direct database URL (postgres://).
   - `max_open_connections` - Can be used to control the maximum number of
     connections that can be opened by the controller.
     The minimum number of connections required is 5.
     Setting this value to 0 will allow the controller to open as many
     connections as needed. This value can be a string representing the max number of connections, can refer to a file
     on disk (file://) from which the number of connections will be read; or an env var (env://) from which the
     number of connections will be read.
-
-  Either URL can refer to a file on disk (file://) from which a URL will be read; an env
-  var (env://) from which the URL will be read; or a direct database URL (postgres://).
+  - `max_idle_connections` - Can be used to control the maximum number of
+    idle connections in the idle connection pool.
+    If `max_open_connections` is greater than 0 but less than
+    `max_idle_connections`, then `max_idle_connections` will be reduced to match the `max_open_connections` limit. 
+    Setting this value to 0 will mean that no idle connections are retained.
+    If not set or set to less than 0, the default
+    [sql.DB](https://pkg.go.dev/database/sql#DB.SetMaxIdleConns) setting will be used.
+    This value can be a string representing the max number of connections, can refer to a file
+    on disk (file://) from which the number of connections will be read; or an env var (env://) from which the
+    number of connections will be read.
```

Comment:
It doesn't look to me like `parseutil.ParsePath` will support a verbatim `"5"`, will it? Do we want
```suggestion
    This value can be an integer representing the max number of connections, can refer to a file
    on disk (file://) from which the number of connections will be read; or an env var (env://) from which the
    number of connections will be read.
```

Reply:
This phrasing was taken from what already existed for `max_open_connections`. I am not sure if `an integer` is more accurate phrasing, but it should be consistent between this and hte `max_open_connections` since they both leverage the same logic.